README.md

```diff
@@ -16,6 +18,8 @@ sudo apt-get install ros-kinetic-costmap-converter
 
 The branch of `ropod-common` should be changed to `develop` branch if `catkin build` fails.
 
+If `catkin build` fails because it can't find `ZyreBaseCommunicator`, execute install instructions of [ropod_common](https://github.com/ropod-project/ropod_common#install).
```

Comment:
"...execute **the** install instructions of ropod_common" or simply "install ropod_common"